chart/values.yaml

```diff
@@ -1116,6 +1116,8 @@ config:
     multi_namespace_mode: '{{ if .Values.multiNamespaceMode }}True{{ else }}False{{ end }}'
 # yamllint enable rule:line-length
 
+# Whether the KubernetesExecutor can launch workers and pods in multiple namespaces
+# If true, it creates ClusterRole/ClusterRolebinding (with access to entire cluster)
```

Comment:
We should also add this second line to `values.schema.json`.

Reply:
Added